specs/filesystem_hierarchy.md

```diff
@@ -0,0 +1,529 @@
+---
+title: FileSystem Hierarchy
+category: Concepts
+layout: default
+version: 1
+SPDX-License-Identifier: CC-BY-4.0
+---
+
+# FileSystem Hierarchy
+
+## Description
+
+Operating systems using the
+[`systemd(1)`](https://www.freedesktop.org/software/systemd/man/systemd.html)
+system and service manager are organized based on a file system hierarchy
+inspired by UNIX, more specifically the hierarchy described in the [File System
+Hierarchy](http://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.html)
+specification and
+[`hier(7)`](https://man7.org/linux/man-pages/man7/hier.7.html), with various
+extensions, partially documented in the [XDG Base Directory
+Specification](https://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html)
+and [XDG User
+Directories](https://www.freedesktop.org/wiki/Software/xdg-user-dirs). This
+manual page describes a more generalized, though minimal and modernized subset
```

Comment:
Not "manual page" now. Actually, that whole paragraph needs to be rewritten. So let's merge this as is, i.e. with unchanged text, and then update later.